README.md

```diff
@@ -9,7 +9,7 @@ Please see the documentation and guide below to get started.
 ## Documentation
```

Comment:
Do we want to explicitly point out that stable users should pick the latest release-mm-yy branch here, to make it more visible? It might be a bit too hidden if only on the release process documentation, which is something users might  rarely look at.

Reply:
I'm not sure that's necessary, there is already a release link in the guide that will show the latest first. And now users can use the main branch instead of a tagged release.

I don't think anyone's had a problem with it before.

Reply:
Ok